kubernetes_deploy/dev-lgfs/deployment.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: claim-for-crown-court-defence
+  namespace: cccd-dev-lgfs
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 100%
+  selector:
+    matchLabels:
+      app: cccd
+  template:
+    metadata:
+      labels:
+        app: cccd
+    spec:
+      containers:
+        - name: cccd-app
+          imagePullPolicy: Always
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-get-paid/cccd:set-me
+          ports:
+            - containerPort: 3000
+          readinessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          env:
+            - name: ENV
+              value: 'dev'
```

Comment:
not sure of impact but this might need to be dev-lgfs too. 

note the hardship feature flag is based on env name at the moment which means you would have to amend it to be enabled on whatever this env is "called"

Reply:
I thought that ENV['ENV'] was always `dev` as apposed to `production` or `test`

Reply:
ENV is the name of the environment `dev`/`staging` etc, . RAILS_ENV is always `production`

Reply:
We could do ```ENV['ENV'].match?(/^dev.*/)```